testing/traefik/APKBUILD

```diff
@@ -0,0 +1,60 @@
+pkgname=traefik
+pkgver=1.7.4
+pkgrel=0
+pkgdesc="The Cloud Native Edge Router"
+url="https://traefik.io"
+arch="all"
+license="MIT"
+makedepends="go go-bindata"
+install="$pkgname.pre-install"
+pkgusers="$pkgname"
+pkggroups="$pkgname"
+subpackages="$pkgname-openrc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/containous/$pkgname/archive/v$pkgver.tar.gz
```

Comment:
please change it to `source="$pkgname-$pkgver.tar.gz::https://github.com/containous/traefik/archive/v$pkgver.tar.gz` to make navigation easy

PS: https://github.com/containous/traefik/releases/tag/v1.7.5 is out

Reply:
Ok I'll update and fix that, thanks

Reply:
done, updated to 1.7.5 also